source/Playnite.DesktopApp/Themes/Desktop/Default/CustomControls/ComboBoxList.xaml

```diff
@@ -101,6 +166,6 @@
     <Style TargetType="{x:Type DdItemListSelectionBox}">
         <Setter Property="SnapsToDevicePixels" Value="True" />
         <Setter Property="Height" Value="24" />
-        <Setter Property="Template" Value="{DynamicResource ComboBoxListBaseTemplate}" />        
+        <Setter Property="Template" Value="{DynamicResource FilterComboBoxListTemplate}" />        

```

Comment:
I don't think this is correct, you are setting template that doesn't have features you added and all other drop down styles now have template with those features...

Reply:
Never mind, it was buggy GitHub diff that showed it flipped.